client/my-sites/plugins-wpcom/plugin-install-button.jsx

```diff
@@ -37,13 +38,23 @@ export const WpcomPluginInstallButton = props => {
 	function installButtonAction( event ) {
 		event.preventDefault();
 
-		const hasErrors = !! get( eligibilityData, 'eligibilityHolds', [] ).length;
-		const hasWarnings = !! get( eligibilityData, 'eligibilityWarnings', [] ).length;
+		const eligibilityHolds = get( eligibilityData, 'eligibilityHolds', [] );
+		const eligibilityWarnings = get( eligibilityData, 'eligibilityWarnings', [] );
+
+		const hasErrors = !! eligibilityHolds.length;
+		const hasWarnings = !! eligibilityWarnings.length;
 
 		if ( ! hasErrors && ! hasWarnings ) {
 			// No need to show eligibility warnings page, initiate transfer immediately
 			initiateTransfer( siteId, null, plugin.slug );
 		} else {
+			props.recordTracksEvent( 'calypso_automatic_transfer_plugin_install_ineligible',
+				{
+					eligibilityHolds: eligibilityHolds.join( ', ' ),
+					eligibilityWarnings: eligibilityWarnings.join( ', ' ),
+					plugin_slug: plugin.slug
+				} );
```

Comment:
it would be nice if instead of importing `recordTracksEvents` directly into the component we instead wrapped `initiateTransfer` with the tracks events so that the component can remain agnostic to the metrics systems